io/shared/src/main/scala/scalaz/effect/Fiber.scala

```diff
@@ -43,8 +41,8 @@ trait Fiber[E, A] { self =>
   /**
    * Monoidal composition of fibers.
    */
-  final def <>(that: => Fiber[E, A])(implicit M: Monoid[A]): Fiber[E, A] =
-    zipWith(that)(M.append(_, _))
+  final def <>(that: => Fiber[E, A])(mappend: (A, A) => A): Fiber[E, A] =
```

Comment:
This should be deleted, the user and existing references can just call `zipWith`, it's more powerful, anyway.

For `<>`, the user will need Scalaz syntax for the monoid instance.

Reply:
Yeah, i thought about deleting it the same way I got rid of the instance resolvers in the companion object.